README.md

```diff
@@ -64,3 +68,32 @@ For more information, see the [Rust API embedding chapter] of the Wasmtime docum
 [Rust API embedding chapter]: https://bytecodealliance.github.io/wasmtime/embed-rust.html
 
 It's Wasmtime.
+
+#### Standalone runtime
+
+The [Releases](https://github.com/bytecodealliance/wasmtime/releases) page has pre-built binaries for each release, with binaries for Linux, macOS and Windows.
+
+If you want to build the latest Wasmtime from scratch, you will need:
+
+* An updated [Rust toolchain](https://www.rust-lang.org/install.html) installed
+* A C toolchain installed - e.g. the [`build-essential`](https://packages.ubuntu.com/xenial/build-essential) package in Ubuntu
+* A clone of the Wasmtime Git repo
+
+Example:
+
+```sh
+git clone --recursive https://github.com/bytecodealliance/wasmtime.git
+cargo build --release
+```
+
+This will create a binary at `./target/release/wasmtime`, which you can then invoke directly to run a given WASI-compatible WASM binary:
+
+```sh
+$ ./target/release/wasmtime YOUR-BINARY.wasm
+```
+
+See `wasmtime --help` for the full list of options.
+
+### Licence
+
+Apache 2.0 licensed. See the LICENSE file for details.
```

Comment:
I think that if we want to call out the license here we probably want to also mention that it has the standard "LLVM exception" as well? Others may know better for sure though how to write down what license we have

Reply:
Do we want to call out the license here? Github already detects the license and displays a license badge on the [main page](https://github.com/bytecodealliance/wasmtime/), and a `LICENSE` file at the top level is a widely recognized convention.